tests/unit/commands/local/invoke/test_cli.py

```diff
@@ -28,6 +28,7 @@ def setUp(self):
         self.debug_ports = [123]
         self.debug_args = "args"
         self.debugger_path = "/test/path"
+        self.additional_volumes = [Path.home()]
```

Comment:
If this feature supports multiple additional volumes, can we explicitly test that?

Reply:
@awood45, updated `test_cli.py` with multiple volumes by default. I've added some tests to verify cases when there are multiple volumes in `tests/unit/local/docker/test_lambda_container.py`.